src/test/scala/io/archivesunleashed/ArchiveRecordTest.scala

```diff
@@ -92,7 +92,7 @@ class ArchiveRecordTest extends FunSuite with BeforeAndAfter {
       "http://www.archive.org/robots.txt", "http://www.archive.org/").deep)
   }
 
-  test("Mime-Type") {
+  test("MIMEtype") {
```

Comment:
does it matter that in line 101 and 102 it's shown as exampleMimeType ? (T is capitalized

Reply:
We can discard this comment.... noticed this was the pattern throughout :)